corehq/apps/style/static/style/js/hq.helpers.js

```diff
@@ -136,3 +136,15 @@ $.fn.koApplyBindings = function (context) {
     }
     return ko.applyBindings(context, this.get(0));
 };
+
+var myobj = {
+    'test': 3,
+    'foo': 'bar'
+};
+
+var myobj2 = {
+    'test': 3,
+    'foo': 'bar',
+};
+
+var myobj3 = {'test': 3, 'foo': 'bar',};
```

Comment:
'myobj3' is defined but never used (no-unused-vars)
Unexpected trailing comma. (comma-dangle)